src/main/java/org/folio/circulation/domain/CheckInContext.java

```diff
@@ -237,4 +241,23 @@ public ItemStatus getItemStatusBeforeCheckIn() {
   public boolean areLostItemFeesRefundedOrCancelled() {
     return lostItemFeesRefundedOrCancelled;
   }
+
+  public JsonObject asJson() {
```

Comment:
Maybe we should extract this logic to a mapper class. e.g. `CirculationCheckInLogEventMapper`?